lib/elixir/test/elixir/keyword_test.exs

```diff
@@ -42,4 +42,35 @@ defmodule KeywordTest do
       Keyword.get_and_update!([a: 1], :a, fn value -> value end)
     end
   end
+
+  test "merge/2" do
```

Comment:
@eksperimental I meant that every other function in `Keyword` is doctested, so it feels weird to only have separate tests for `merge/2`.